ext/opcache/ZendAccelerator.c

```diff
@@ -84,6 +84,9 @@ typedef int gid_t;
 #include <immintrin.h>
 #endif
 
+// TODO: remove ???
+#include <sys/mman.h>
```

Comment:
I know its a WIP but this part (along with the mmap call) won't work for Windows and it needs to be using the File Mapping API (hence the AppVeyor fail)